style/styles.css

```diff
@@ -12,6 +12,58 @@ body {
   background: #4D469F url('https://thumbs.gfycat.com/EvenScaryKitfox-size_restricted.gif');
 }
 
+.pop-up-screen {
+  z-index: 1; 
+  width: 1440px;
+  height: 820px;
+  position: absolute;
```

Comment:
The screen looks really good. Does width and height need to be this specific for absolute positions?